modules/ROOT/pages/to-request-access.adoc

```diff
@@ -26,19 +26,19 @@ If terms and conditions are set for an asset, you are prompted to accept the
 terms before access is granted, and the created contract includes the terms and
 conditions.
 
-. Click an API asset to view more details.
+. Click an API or API Group asset to view more details.
 . Click *Request Access*.
-. Choose the API instance from the list.
+. Choose the API instance or API Group instance from the list.
 . If you are using an existing application, choose from the list of applications.
 +
-An application and its API instance use the same client provider. If an organization administrator has configured xref:api-manager::configure-multiple-credential-providers.adoc[multiple client providers] in Anypoint API Manager, then the application list shows only the applications that use the same client provider as the selected API instance.
+An application and its asset instance use the same client provider. If an organization administrator has configured xref:api-manager::configure-multiple-credential-providers.adoc[multiple client providers] in Anypoint API Manager, then the application list shows only the applications that use the same client provider as the selected asset instance.
 . If you do not have an application to which to provide access, click *Select application > Create a new application*. 
 +
 See the section "Create a New Application" for more information.
 . Choose the service-level agreement (SLA) value for the application.
 . Click Request Access.
 +
-The Request API Access menu opens to show that the API access was successful. This screen lists the client ID and client secret for this API access instance.
+The Request API Access menu opens to show that the asset access was successful. This screen lists the client ID and client secret for this asset access instance.
```

Comment:
"show that the ~asset~ access was ~succesful~ automatically granted or submitted for review"

Reply:
Fixed.